doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -354,7 +354,7 @@ Protocol serialization samples
 SDFW samples
 ------------
 
-|no_changes_yet_note|
+* Removed the ``ssf_client`` sample since all services being shown has been removed.
```

Comment:
```suggestion
* Removed the SDFW: Service Framework Client sample since all related services are removed.
```

Reply:
Not so sure about the sample removal just like that. Shouldn't it first be stated as deprecated and removed only in later release?